<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ClearReceiveBuffer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>清除接收</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制显示</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendPushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReceiveStop">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SavePushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存窗口</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoSend">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>定时发送</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AutoSendTime">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearSendDataPushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>320</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_timer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_connect_state">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接状态</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ReceiveDatatextBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommunic">
    <property name="title">
     <string>Communication(&amp;C)</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconncet"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommunic"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionDisconncet"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconncet">
   <property name="text">
    <string>Disconncet</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
